### PR TITLE
Pmp prp mandatory integration

### PR DESCRIPTION
**Pull request by reinbach**
[ch17422]

### CODE REVIEW

File: src/etools/applications/partners/validation/interventions.py [hunk 1]
```diff
@@ -168,6 +168,18 @@ def transition_to_active(i):
         raise TransitionError([
             _('PD cannot be activated if the Partner is Blocked in Vision')
         ])
+
+    # PD/SSFA require results framework and reporting requirements
+    if i.document_type in [i.PD, i.SSFA]:
```
Comment (robertavram) [inline src/etools/applications/partners/validation/interventions.py]: everything here.. not just PD, SSFA.. SHPD is actually just a type of PD

Comment (reinbach) [reply src/etools/applications/partners/validation/interventions.py]: :+1:

File: src/etools/applications/partners/validation/interventions.py [hunk 2]
```diff
@@ -169,8 +169,8 @@ def transition_to_active(i):
             _('PD cannot be activated if the Partner is Blocked in Vision')
         ])
 
-    # PD/SSFA require results framework and reporting requirements
-    if i.document_type in [i.PD, i.SSFA]:
+    # PD/SHPD/SSFA require results framework and reporting requirements
+    if i.document_type in [i.PD, i.SSFA, i.SHPD]:
```
Comment (robertavram) [inline src/etools/applications/partners/validation/interventions.py]: this will be true every time :) can be removed.

Comment (reinbach) [reply src/etools/applications/partners/validation/interventions.py]: you say that now, until we add a new type that doesn't belong!! :)

**Pull request merged**